android/KMEA/app/src/main/java/com/tavultesoft/kmea/ConfirmDialogFragment.java

```diff
@@ -6,22 +6,35 @@
 import android.app.DialogFragment;
 import android.content.DialogInterface;
 import android.os.Bundle;
+import android.view.Window;
 import android.widget.Toast;
 
 /**
  * Confirmation dialog for downloading a Keyman keyboard
  */
 public class ConfirmDialogFragment extends DialogFragment {
-  public static String ARG_TITLE = "ConfirmDialogFragment.title";
+  public static final String ARG_TITLE = "ConfirmDialogFragment.title";
+  public static final String ARG_MESSAGE = "ConfirmationDialogFragment.message";
```

Comment:
Can we give this the same name `ConfirmDialogFragment.message` rather than `ConfirmationDialogFragment.message`?